main_test.go

```diff
@@ -39,19 +34,45 @@ func TestRegex(t *testing.T) {
 	}
 
 	for _, path := range paths {
-		AssertThat(t, staticRewrite(path), NotContains{"log"})
-		AssertThat(t, staticRewrite(path), NotContains{"vnc"})
+		AssertThat(t, staticRewrite(path), Not{Contains{"log"}})
```

Comment:
it's not what you think. It checks presence in the slice in this form. Return back please